samples/msgext-doc-compliance-checker/appPackage/manifest.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.16",
+  "version": "1.0.0",
+  "id": "${{TEAMS_APP_ID}}",
+  "packageName": "com.microsoft.teams.extension",
+  "developer": {
+    "name": "Teams App, Inc.",
+    "websiteUrl": "https://www.example.com",
+    "privacyUrl": "https://www.example.com/privacy",
+    "termsOfUseUrl": "https://www.example.com/termofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "msgext-doc-compliance-checker",
+    "full": "full name for msgext-doc-compliance-checker"
+  },
+  "description": {
+    "short": "short description of msgext-doc-compliance-checker",
```

Comment:
Add these names and short, full descriptions.
If you are not sure, check with Anshul or Sathya.

Reply:
Fixed